src/sidebar/util/test/time-test.js

```diff
@@ -77,15 +77,15 @@ describe('sidebar.util.time', function() {
     it('Handles empty dates', function() {
       const t = null;
       const expect = '';
-      assert.equal(time.toFuzzyString(t, mockIntl()), expect);
+      assert.equal(time.toFuzzyString(t, undefined, mockIntl()), expect);
     });
 
     const testFixture = function(f) {
       return function() {
         const t = new Date().toISOString();
         const expect = f[1];
         sandbox.clock.tick(f[0] * 1000);
-        assert.equal(time.toFuzzyString(t, mockIntl()), expect);
+        assert.equal(time.toFuzzyString(t, undefined, mockIntl()), expect);
```

Comment:
I find this really hard to follow and have a couple thoughts about how to fix that. Moving the test cases to the location where they are used would be helpful here in terms of readability, also passing the params explicitly by name here instead of using f[0], f[1]. If we pass in the time to the toFuzzyString rather than undefined we won't have to tick the clock and this test will be much easier to read IMO.